tests/analyse/test_scenario.py

```diff
@@ -1602,6 +1602,33 @@ def test_scenario_with_satellite_data():
     xr.testing.assert_equal(model_scenario.footprint.data, footprint.data)
 
 
+def test_model_scenario_col_fp_data_merge():
+    """This is to test satellite data aligns fp to obs and satifies the fp_data_merge functionality"""
+
+    obs_column_data = get_obs_column(species="ch4", max_level=3,
+                     satellite="gosat", domain='southamerica', store="group")
```

Comment:
As above this should be updated to:
```suggestion
                     satellite="gosat", obs_region='brazil', store="group")
```